test/hotspot/jtreg/runtime/Safepoint/TestAbortOnVMOperationTimeout.java

```diff
@@ -51,11 +51,9 @@ public static void main(String[] args) throws Exception {
             return;
         }
 
-        // These should definitely pass: more than a minute is enough for Serial to act.
+        // These should definitely pass: more than 3 minutes is enough for Serial to act.
```

Comment:
"These" is no longer accurate since this has been changed to test a single value.

Reply:
Fixed.